vmdb/app/models/vm_openstack.rb

```diff
@@ -17,4 +17,57 @@ def self.calculate_power_state(raw_power_state)
     else                                                   super
     end
   end
+
+  def perform_metadata_scan(ost)
+    require 'OpenStackExtract/MiqOpenStackVm/MiqOpenStackInstance'
+
+    log_pref = "MIQ(#{self.class.name}##{__method__})"
+
+    instance_id = ems_ref
```

Comment:
Actually, after the rubocop cleanup, there's no reason for it. I'll clean it up.